apps/E2E/src/Checkbox/specs/Checkbox.spec.win.ts

```diff
@@ -52,31 +68,36 @@ describe('Checkbox Functional Testing', () => {
 
   it("Click on a Checkbox -> Validate it toggles correctly AND calls the user's onChange", async () => {
     /* Validate the Checkbox is initially toggled OFF */
-    await expect(await CheckboxPageObject.isCheckboxChecked()).toBeFalsy();
+    await expect(await CheckboxPageObject.isCheckboxChecked()).toBeFalsy('The primary checkbox should initially be toggled off.');
 
     /* Click on the Checkbox to toggle on */
-    await CheckboxPageObject.clickComponent();
+    await CheckboxPageObject.click(CheckboxPageObject._primaryComponent);
     await CheckboxPageObject.waitForCheckboxChecked(PAGE_TIMEOUT);
 
-    await expect(await CheckboxPageObject.didOnChangeCallbackFire()).toBeTruthy();
+    await expect(await CheckboxPageObject.didOnChangeCallbackFire()).toBeTruthy(
+      'Primary checkbox failed to fire onChange() callback via click.',
+    );
 
     /* Validate the Checkbox is toggled ON */
-    await expect(await CheckboxPageObject.isCheckboxChecked()).toBeTruthy();
+    await expect(await CheckboxPageObject.isCheckboxChecked()).toBeTruthy('The primary checkbox should have been toggled on.');
 
     await CheckboxPageObject.clickComponent();
 
     /* Validate the Checkbox is toggled OFF */
-    await expect(await CheckboxPageObject.isCheckboxChecked()).toBeFalsy();
+    await expect(await CheckboxPageObject.isCheckboxChecked()).toBeFalsy('The primary checkbox should have been toggled off.');
+
     await expect(await CheckboxPageObject.didAssertPopup()).toBeFalsy(CheckboxPageObject.ERRORMESSAGE_ASSERT);
   });
 
   it('Click the "SPACE" on a Checkbox and verify it toggles', async () => {
```

Comment:
I think this should be renamed "Press "Space" on a Checkbox...." instead of "Click the..."